crates/topos-tce-broadcast/src/task_manager/mod.rs

```diff
@@ -136,9 +146,12 @@ impl TaskManager {
                         debug!("Task for certificate {} finished successfully", certificate_id);
                         self.tasks.remove(&certificate_id);
                         DOUBLE_ECHO_ACTIVE_TASKS_COUNT.dec();
+
                     } else {
                         debug!("Task for certificate {} finished unsuccessfully", certificate_id);
                     }
+
+                    self.next_pending_certificate();
```

Comment:
So this is the key addition in the PR? What is the thinking here?

Reply:
When a `task` finish, we ask for more work instead of waiting the next tick.

Reply:
Do you think we need to look into having shorter ticks, the default 100ms is quite long and afaik we have CPU capacity to spare in production? (Not for this PR of course, but it could be useful to expose in the config).